### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by kovacshuni**

### CODE REVIEW

File: app.go [hunk 1]
```diff
@@ -7,23 +7,25 @@ import (
...
+	config   *notifierConfig
 	client   *http.Client
 }
 
-type config struct {
+type notifierConfig struct {
 	mappingURL              string
 	cmsMetadataNotifierAddr string
 	cmsMetadataNotifierHost string
+	cmsMetadataNotifierAuth string
```
Comment (Nesze) [inline app.go:23]: is that needed?

File: handler.go [hunk 2]
```diff
@@ -100,7 +100,12 @@ func (mm metadataMapper) sendMetadata(metadata []byte, tid string) error {
 		return fmt.Errorf("Creating request: [%v]", err)
 	}
 	req.Header.Add("Content-type", "application/json")
-	req.Host = mm.config.cmsMetadataNotifierHost
+	if mm.config.cmsMetadataNotifierHost != "" {
```
Comment (Nesze) [inline handler.go]: there is a sane default set to this value

File: app.go (hunk 1 above)
Comment (kovacshuni) [inline app.go:23]: not necessarily, but it may help for debugging. I left this here.

File: handler.go (hunk 2 above)
Comment (kovacshuni) [inline handler.go]: removed. :)

**Comment by Nesze** [comment]
👍

**Pull request merged**